test/controllers/library_entries_controller_test.rb

```diff
@@ -79,7 +79,7 @@ class LibraryEntriesControllerTest < ActionController::TestCase
   end
 
   test "need to be authenticated as correct user to update library entry" do
-    id = watchlists(:one).id
+    id = library_entries(:one).id
```

Comment:
I have better things to do with my time than refactor already-working tests :wink: